### ISSUE TITLE
Update format of the report scores (`Properties:` line is confusing)

### ISSUE THREAD

**Issue by npatki** [sdv-dev/SDMetrics#297]
### Problem Description

Currently, the `QualityReport` prints out the following:

```
Creating report: 100%|██████████| 4/4 [00:00<00:00,  7.09it/s]

Overall Quality Score: 82.84%

Properties:
Column Shapes: 82.78%
Column Pair Trends: 82.9%
```

We intend for `Properties:` to be the title, with `Column Shapes` and `Column Pair Trends` the sub-bullets. However, this is unclear and users may wonder why `Properties:` doesn't have a value next to it.

### Expected behavior

Add bullets to the items that appear under `Properties:` to make the hierarchy more explicit.

```
Creating report: 100%|██████████| 4/4 [00:00<00:00,  7.09it/s]

Overall Quality Score: 82.84%

Properties:
- Column Shapes: 82.78%
- Column Pair Trends: 82.9%
```

**Comment by Nordine67200** [comment]
Hello,

My user is not allowed to create the branch when pushing the modification (push --set-upstream origin update_format_report_score). I cannot ask for a pull request.

Was there already a branch that i should use to push the modifications?

**Comment by npatki** [comment]
Hi @Nordine67200, hmm seems like your problem is unrelated to the the main topic (`Properties:` in the report). 

Would you be able to [file a new issue](https://github.com/sdv-dev/SDMetrics/issues/new/choose) so we can converse there?

**Comment by npatki** [comment]
This issue was addressed in PR #377 and will be updated in the next SDMetrics release. I'm closing off in favor of that.
